packages/modal/storybook/Modal.js

```diff
@@ -115,3 +115,10 @@ storiesOf('Modal', module)
       verticalAlignment="top"
     />
   ))
+  .add('Zero Timeout', () => (
```

Comment:
If the point of this is to remove the animation of the transition, we may want to make the story more descriptive, like, "Use 0 timeout to open/close dialog without animated transition"

Reply:
The animation still happens but there is no delay before it starts, that's what timeout does